SBAT.md

```diff
@@ -140,29 +140,39 @@ value associated with the data currently stored in that variable.
 
 #### Generation-Based Revocation Scenarios
 
-Products (**not** vendors, a vendor can have multiple products or even pass a
+**Products** (**not** vendors, a vendor can have multiple products or even pass a
 product from one vendor to another over time) are assigned a name. Product
 names can specify a specific version or refer to the entire product family. For
 example mydistro and mydistro,12.
 
-Components that are used as a link in the UEFI Secure Boot chain of trust are
+**Components** that are used as a link in the UEFI Secure Boot chain of trust are
 assigned names. Examples of components are shim, GRUB, kernel, hypervisors, etc.
 
+Below is an example of a product and vendor, both in the same `sbat.csv` file. SBAT
+is a vendor, as it is a link in the UEFI Secure Boot chain of trust.
+`grub.acme` is an example of a product name.
+```
+sbat,1,SBAT Version,sbat,1,https://github.com/rhboot/shim/blob/main/SBAT.md
+grub.acme,1,Acme Corporation,grub,1.96-8191,https://acme.arpa/packages/grub
+```
+
 We could conceivably support sub-components, but it's hard to conceive of a
 scenario that would trigger a UEFI variable update that wouldn't justify a
 hypervisor or kernel re-release to enforce that sub-component level from there.
 Something like a "level 1.5 hypervisor" that can exist between different kernel
 generations can be considered its own component.
 
-Each component is assigned a minimum global generation number. Vendors signing
+Each component is assigned a **minimum global generation number**. Vendors signing
 component binary artifacts with a specific global generation number are
 required to include fixes for any public or pre-disclosed issue required for
 that generation. Additionally, in the event that a bypass only manifests in a
 specific product's component, vendors may ask for a product-specific generation
 number to be published for one of their product's components. This avoids
 triggering an industry wide re-publishing of otherwise safe components.
 
-A product-specific minimum generation number only applies to the instance of
+In the example above, 1 is sbat's minimum global generation number.
```

Comment:
I think we've been referring to the sbat "generation" as the version, but I'm not sure that distinction doesn't cause more confusion than it helps. The rest of the doc should be scanned and either fixed up globally of this should use the term version as well.

Reply:
I'm happy to do that, but I'd like you to make the call on the term. In other places, I've been seeing "version-number" and "global number". (I'm assuming global number is a subcategory of version-number)

Reply:
I guess this is slightly confusing:

The number on the sbat line is the version of the format of the sbat metadata. We may or may not ever need to change that. We'll certainly try hard to extend it in a backwards compatible way. It doesn't really revoke anything.

There is a global generation number for a component which needs to be satisfied by any instance of that component. Non-global generation numbers are the product specific ones that allow a component to be revoked at some generation for that product only.

This can be separated out from this PR though.